FHIR-Converter/README.md

```diff
@@ -13,69 +13,98 @@ This extension provides an interactive editing and verification experience to cr
 ## Getting started
 
 After you have installed the extension, follow these steps to edit the templates:
-
-1. Obtain [sample HL7v2 data files](https://github.com/microsoft/FHIR-Converter/tree/dotliquid/data/SampleData), and put it in a local folder.
-1. Fetch [default templates](https://github.com/microsoft/FHIR-Converter/tree/dotliquid/data/Templates) published by Microsoft and store it in another folder.
-1. Go to VS Code, and create a new converter workspace by pressing CTRL+W. Select the template and the data folder during the process.
-1. Select root template file and the test data file using context menu. 
-1. Convert data by pressing CTRL+R or context menu. After converting, the extension should open a 3-pane view having data, template, and converted FHIR bundle in the views.
-1. Edit template, and/or data files and press CTRL+R to refresh the output. The output pane should get refreshed and highlight changes in the output.
-1. Save the changed templates for subsequent use.
+1. Prepare your own templates or pull templates published by Microsoft and store them in a folder;
+2. Prepare your own data or pull sample data and store them in another folder;
+3. Create a new converter workspace by pressing CTRL+W. Select the root template folder and the data folder during the process;
+4. Select template file and test data file by commands in the context menu; 
+5. Convert data by pressing CTRL+R or by the command in the context menu. After converting, the extension should open a 3-pane view including data, template and converted FHIR bundle in the panes;
+6. Edit template and/or data files and save them. Press CTRL+R to refresh the output. The output pane should get refreshed and highlight the changes in the output;
+7. Save and push the changed templates to your private Azure Container Registry.
 
 See relevant service documentation for using the templates in data conversion process: 
 
 1. [FHIR Converter command line tool](https://github.com/microsoft/FHIR-Converter#command-line-tool) in FHIR Converter OSS project.
-1. [$convert-data](https://github.com/microsoft/fhir-server/blob/master/docs/ConvertDataOperation.md) operation in FHIR Server for Azure.
+2. [$convert-data](https://github.com/microsoft/fhir-server/blob/master/docs/ConvertDataOperation.md) operation in FHIR Server for Azure.
 
 
 ## How to guide
 
-### 1. Create a converter workspace
+### 1. Login to ACR (Azure Container Registry) 
+
+Before pulling and pushing templates using private ACR, you need to confirm whether you have logged in to your ACR. In the template management tool, it provides several ways to log in. Currently, we provide the [oras tool](https://github.com/deislabs/oras) to help your log in quickly without installing other tools. 
+
+You can use the command `FHIR Converter: Login to Azure Container Registry (Ctrl + I)`:
+- First, you need to enter the private ACR `<registry-name>.azurecr.io`, and then the login command will run in the terminal using oras tool;
```

Comment:
Type in the name of your ACR `<registry-name>.azurecr.io`, and press enter. It will open the terminal and run the oras tool.